helm/Chart.yaml

```diff
@@ -15,13 +15,13 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: "1.3.5"
+version: "v1.3.5"
```

Comment:
I think the `version` field has to be SemVer compliant. It looks like the helm checks didn't run for this one -- I think it'll fail to publish the chart with a non-SemVer version

Reply:
@bstrdsmkr Thanks for the catch.
@downiec We'll either need to separate `version` from `appVersion` as I'd mentioned before or change how we're tagging the container images e.g. move from v1.3.5 to 1.3.5. I think the former is probably a better option, but it's up to you.